### DISCUSSION TITLE
Língua inglesa dos escravos

### DISCUSSION THREAD

**Discussion by KeithBueno** [frontendbr/forum#2501] [Perguntas e Respostas]
Alguém poderia me indicar filmes sobre a escravidão americana que retrate com fidelidade como os negros falavam o inglês nessa época?
Estou ensaiando para um concerto e a maior parte das peças são Spirituals e gostaria de entender a pronúncia do idioma nesta época.

**Comment by Bielduceeu** [comment]
Ja ouvi falar sobre o Old-English, conhecido com anglo-saxão. Joga no GPT, acho que não é o que procura, mas vai dar auxilio.
